dotnet-sourcelink-shared/SourceLinkMap.cs

```diff
@@ -0,0 +1,198 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+// TODO: Use a source package once available (https://github.com/dotnet/sourcelink/issues/443)
+
+namespace Microsoft.SourceLink.Tools
```

Comment:
nit: Should this file include a comment pointing toward the source in the dotnet/sourcelink repo?